lib/absinthe/type/built_ins/scalars.ex

```diff
@@ -59,6 +59,48 @@ defmodule Absinthe.Type.BuiltIns.Scalars do
     parse parse_with([Absinthe.Blueprint.Input.Boolean], &parse_boolean/1)
   end
 
+  scalar :utc_datetime, name: "DateTime" do
```

Comment:
Was there a specific reason to need to opt-in to timezone? I know for me, I prefer to default to being overly specific which would be using a timezone by default. This also goes against the assumptions of the Elixir conventions of DateTime is specific while NaiveDateTime is not specific. Is there any reason not to use `:naive_datetime` as the identifier to keep that consistency?